### PR TITLE
Added cypress testcase to verify host management page [IHARAR-9998]

### PR DESCRIPTION
**Pull request by binalee21**

### CODE REVIEW

File: cypress/integration/verifyHostManagementElements.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        headerElements.forEach((headerElement) => {
+            cy.verifyElementWithText('th', headerElement);
+        });
+
+        //fetch last hostTitle text from the hostManagement list
+        cy.get('.ant-table-row td:nth-child(2)')
+            .last()
+            .then( (hostTitle) => {
+                searchHostTitle = hostTitle.text();
+                cy.log(searchHostTitle);
```
Comment (Shreya76) [inline cypress/integration/verifyHostManagementElements.js:39]: Remove this line.

File: cypress/support/commands.js [hunk 2]
```diff
@@ -208,6 +208,41 @@ Cypress.Commands.add('dragList',(dragSelector, dropSelector ) => {
   .click({force: true})
 });
 
+Cypress.Commands.add('verifyElementWithText',(parentTag, elementText) => {
+  cy.get(parentTag)
+    .contains(elementText)
+    .should('be.visible');
+});
+
+Cypress.Commands.add('getRandomString',(stringLength, variable) => {
```
Comment (Shreya76) [inline cypress/support/commands.js:217]: Remove this method.

File: cypress/integration/verifyHostManagementElements.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+describe('Verify Host Management page along with pagination', () => {
+    let searchHostTitle = "";
```
Comment (Shreya76) [inline cypress/integration/verifyHostManagementElements.js:2]: Let's rename the variable to SEARCH_HOST_TITLE

File: cypress/integration/verifyHostManagementElements.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    let headerElements = [
```
Comment (Shreya76) [inline cypress/integration/verifyHostManagementElements.js:3]: Move this to constants.json

File: cypress/support/commands.js (hunk 2 above)
Comment (binalee21) [reply cypress/support/commands.js]: Done

File: cypress/integration/verifyHostManagementElements.js (hunk 3 above)
Comment (binalee21) [reply cypress/integration/verifyHostManagementElements.js]: Done

File: cypress/integration/verifyHostManagementElements.js (hunk 4 above)
Comment (binalee21) [reply cypress/integration/verifyHostManagementElements.js]: Done

File: cypress/fixtures/constants.json [hunk 5]
```diff
@@ -7,5 +7,14 @@
     "countries": ["AU", "CA"],
     "exclusions": "MX",
     "playlistCategories": "relaxing", 
-    "tracksSearch": ["Relax", "Pursuit"]    
+    "tracksSearch": ["Relax", "Pursuit"],
+    "headerElements":[
```
Comment (Shreya76) [inline cypress/fixtures/constants.json:11]: Please give a space before the [

File: cypress/integration/verifyHostManagementElements.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+        //verifying hostManagement content Header elements
+        constant.headerElements.forEach((headerElement) => {
+            cy.verifyElementWithText('th', headerElement);
+        });
+
+        //fetch last hostTitle text from the hostManagement list
+        cy.get('.ant-table-row td:nth-child(2)')
+            .last()
+            .then( (hostTitle) => {
```
Comment (Shreya76) [inline cypress/integration/verifyHostManagementElements.js:30]: remove unwanted space before (hostTitle)

File: cypress/fixtures/constants.json (hunk 5 above)
Comment (binalee21) [reply cypress/fixtures/constants.json]: Done

File: cypress/integration/verifyHostManagementElements.js (hunk 6 above)
Comment (binalee21) [reply cypress/integration/verifyHostManagementElements.js]: Done

**Pull request merged**